.cicd/scripts/srw_test.sh

```diff
@@ -30,17 +30,25 @@ we2e_experiment_base_dir="${workspace}/expt_dirs"
 we2e_test_dir="${workspace}/tests/WE2E"
 nco_dir="${workspace}/nco_dirs"
 
+# Clean any stale test logs
+rm -f ${workspace}/tests/WE2E/log.*
+rm -f ${we2e_experiment_base_dir}/*/log.generate_FV3LAM_wflow ${we2e_experiment_base_dir}/*/log/* WE2E_summary*txt
+
 # Run the end-to-end tests.
 if "${SRW_WE2E_COMPREHENSIVE_TESTS}"; then
     test_type="comprehensive"
 else
-    test_type="coverage"
+    test_type=${SRW_WE2E_SINGLE_TEST:-"coverage"}
+    if [[ "${SRW_WE2E_SINGLE_TEST}" = skill-score ]]; then
```

Comment:
Since `test_type` is being set to `${SRW_WE2E_SINGLE_TEST}` in the line above (line 41), should `{SRW_WE2E_SINGLE_TEST}` be changed to `test_type`?

```suggestion
    if [[ "${test_type}" = skill-score ]]; then
```

Reply:
The name is legacy from other SRW jobs.  I can look to see about changing this in the scripts and the Jenkinsfile.

Reply:
The `test_type` variable is fine.  I'm just concerned with the `SRW_WE2E_SINGLE_TEST` variable which was added in this PR.  By changing line 42 from `SRW_WE2E_SINGLE_TEST` to `test_type`, I think that manual runs of the Jenkins scripts outside of Jenkins, as well as the automatic submission of the Jenkins tests without manually launching through the API, will successfully pass.

Reply:
Updated Jenkinsfile to use default TASK_DEPTH=9

Also, looking at how wrapper_srw_ftest.sh calls srw_ftest.sh, it already handles empty TASK_DEPTH value, and sets it to '9'.
There is currently no complaint about unbound variable.

Reply:
Updated srw_test.sh according to your suggestion of line 42